<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prest_lib</class>
 <widget class="QDialog" name="prest_lib">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anagrafica rientri e prestiti</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="2">
    <widget class="QTableView" name="tableView">
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="aggiorna_db">
       <property name="text">
        <string>Aggiorna</string>
       </property>
       <property name="icon">
        <iconset resource="../biblio.qrc">
         <normaloff>:/images/view-refresh.png</normaloff>:/images/view-refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancella">
       <property name="text">
        <string>Elimina</string>
       </property>
       <property name="icon">
        <iconset resource="../biblio.qrc">
         <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stampa">
       <property name="text">
        <string>Stampa</string>
       </property>
       <property name="icon">
        <iconset resource="../biblio.qrc">
         <normaloff>:/images/document-print.png</normaloff>:/images/document-print.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="esci">
       <property name="text">
        <string>Chiudi</string>
       </property>
       <property name="icon">
        <iconset resource="../../initdb/images.qrc">
         <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="1">
       <widget class="QLineEdit" name="cerca_txt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Cerca</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="_search">
        <property name="text">
         <string>Cerca</string>
        </property>
        <property name="icon">
         <iconset resource="../biblio.qrc">
          <normaloff>:/images/edit-find.png</normaloff>:/images/edit-find.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Cerca per alunno</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cerca per libro prestato o non prestato</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="id"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Alunno</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="cliente"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Libro</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="libro"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Data prestito</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Data rientro</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="checkBox">
          <property name="whatsThis">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="lib_rientro">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDateEdit" name="data_prestito"/>
        </item>
        <item row="4" column="1">
         <widget class="QDateEdit" name="data_rientro"/>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="list_r">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Visualizza lista completa</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Visualizza lista libri rientrati</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Inserisci &quot;Rientrato&quot; e &quot;Non rientrato&quot;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>id</tabstop>
  <tabstop>cliente</tabstop>
  <tabstop>libro</tabstop>
  <tabstop>data_prestito</tabstop>
  <tabstop>data_rientro</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>list_r</tabstop>
  <tabstop>lib_rientro</tabstop>
  <tabstop>cerca_txt</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>_search</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>aggiorna_db</tabstop>
  <tabstop>cancella</tabstop>
  <tabstop>stampa</tabstop>
  <tabstop>esci</tabstop>
 </tabstops>
 <resources>
  <include location="../../initdb/images.qrc"/>
  <include location="../biblio.qrc"/>
 </resources>
 <connections/>
</ui>
